### ISSUE TITLE
scheduling the learning rate for sub_imagenet datasets.

### ISSUE THREAD

**Issue by YanJenHuang** [zeiss-microscopy/BSConv#8]
In paper 5.3:
For Stanford Dogs, Stanford Cars, and Oxford 102 Flowers, the learning rate is adjust as below:
"The initial learning rate is set to 0.1
and linearly decayed at every epoch such that it approaches
zero after a total of 100 epochs."

How does the learning rate adjusted? Does it wrote in the script (bsconv_pytorch_train.py)?
Is the learning rate adjusted as below?
epoch 0, lr_rate = 0.1
epoch 1, lr_rate = 0.1-(0.1/100)*1
..
epoch n, lr_rate = 0.1-(0.1/100)*n
..
epoch 99, lr_rate = 0.1-(0.1/100)*99

**Comment by zeiss-mamthor** [comment]
Hi,
yes, this  is correct.
Currently this learning rate schedule is not yet supported by the bsconv_pytorch_train.py, but will be added soon. Thank you for the suggestion.

**Comment by xiaohe725** [comment]
> Hi, yes, this is correct. Currently this learning rate schedule is not yet supported by the bsconv_pytorch_train.py, but will be added soon. Thank you for the suggestion.

May I ask if the specific settings for the Stanford Dogs dataset can be provided? I trained 100epochs, and the learning rate also decayed linearly, but the accuracy decreased by about 5%
